ISPN-6626 - IntegrationTest.testGet* fails on Windows

andyuk1986 opened the pull request:
https://issues.jboss.org/browse/ISPN-6626

Adding check for "\r" which will work in case the test is run on Windows machine.

slaskawi commented:
LGTM

anistor commented on server/rest/src/test/scala/org/infinispan/rest/IntegrationTest.scala:
```diff
@@ -171,8 +171,8 @@ class IntegrationTest extends RestServerTestBase {
       val plain = getCollection("text/plain;charset=UTF-8")
-      assertTrue(plain.contains("a\n"))
-      assertTrue(plain.contains("b\n"))
+      assertTrue(plain.contains("a\n") || plain.contains("a\r\n"))
```
Why not use the platform line separator? ```System.getProperty("line.separator")```

anistor commented:
I know this does fix the test but I would rather go to the root of the issue in rest/Server.scala where the response is generated. The response is inconsistently formatted using both a hardcoded ```\n``` and also ```System.lineSeparator()``` here and there. We should just pick one and stick with it.

andyuk1986 commented:
Thanks @anistor for comment. I completely forgot about System.lineSeparator(), thanks for the hint. 
I have changed and ran the test with lineSeparator. Now it passes. In rest/Server.scala the text/plain response is generated with lineSeparator. The only place where "\n" is used is while generating the XML response. I have changed that as well to use the System.lineSeparator(). Hope it is fine.

anistor commented:
Thanks @andyuk1986 ! Pulling!
